.github/actions/setup-ci/action.yml

```diff
@@ -1,25 +1,35 @@
 name: 'Setup CI'
 description: 'Sets up runner for CI'
+inputs:
+  go_cache:
+    description: 'Set up Go cache'
+    default: 'true'
+  elixir_cache:
+    description: 'Set up Elixir cache'
+    default: 'true'
```

Comment:
Allow skipping the cache pull. The Go cache is pretty big and takes ~45s to pull.